tests/__init__.py

```diff
@@ -112,3 +113,11 @@ def get_auth_header(real_header=True, authorized=True, group='hca', email=True,
         info = UNAUTHORIZED_GCP_CREDENTIALS
     token = get_service_jwt(info, group, email=email, email_claim=email_claim) if real_header else str(uuid.uuid4())
     return {"Authorization": f"Bearer {token}"}
+
+
+def needs_terraform(test_item):
```

Comment:
Terraform is a requirement for working with the DSS. Shouldn't these tests fail if it is not installed?

Reply:
Travis doesn't have terraform.  I'll change it to only skip on travis since that's the only bogus use-case.